content/en/docs/migrating/_index.md

```diff
@@ -0,0 +1,835 @@
+})
+```
+
+###### V2
+
+```go
+import "github.com/aws/aws-sdk-go-v2/config"
+import "github.com/aws/aws-sdk-go-v2/service/s3"
+
+// ...
+
+cfg, err := config.LoadDefaultConfig()
+if err != nil {
+	// handle error
+}
+
+client := s3.NewFromConfig(cfg, func(o *s3.Options) {
+	o.Region = "us-west-2"
+})
+```
+
+### Endpoints
+
+The [endpoints]({{< apiref v1="aws/endpoints" >}}) package no longer exists in the {{% alias sdk-go %}}. Each service
+client now embeds its required AWS endpoint metadata within the client package. This reduces the overall binary size of
+compiled applications by no longer including endpoint metadata for services not used by your application.
+
+By default, service clients use their configured AWS Region to resolve the service endpoint for the target Region. If
+your application requires a custom endpoint to be specified for a particular service and region, you can specify
+a custom [aws.EndpointResolver]({{< apiref "aws#EndpointResolver" >}}) using the `EndpointResolver` field on the
+`aws.Config` structure. If your application implements a custom
+[endpoints.Resolver]({{< apiref v1="aws/endpoints#Resolver" >}}) you must migrate it to conform to the
+`aws.EndpointResolver` interface. [aws.EndpointResolverFunc]({{< apiref "aws#EndpointResolverFunc" >}}) is provided as
+a convenient way to wrap a resolver function to satisfy the `aws.EndpointResolver` interface.
+
+For more information on endpoints and implementing a custom resolver, see
+[Configuring Client Endpoints]({{% ref "/docs/configuring-sdk/endpoints.md" %}}).
+
+### Invoking API Operations
+
+The number of service client operations methods have been reduced significantly. The `<OperationName>Request`,
+`<OperationName>dWithContext`, and `<OperationName>` methods have all been consolidated into a single operation method.
+
+#### Example
+
+The following example shows how calls to the {{% alias service=s3 %}} PutObject operation would be migrated from
+AWS SDK for Go to {{% alias sdk-go %}}.
+
+##### V1
+```go
+import "context"
+import "github.com/aws/aws-sdk-go/service/s3"
+
+// ...
+
+client := s3.New(sess)
+
+// Pattern 1
+output, err := client.PutObject(&s3.PutObjectInput{
+	// input parameters
+})
+
+// Pattern 2
+output, err := client.PutObjectWithContext(context.TODO(), &s3.PutObjectInput{
+    // input parameters
+})
+
+// Pattern 3
+req, output := client.PutObjectRequest(context.TODO(), &s3.PutObjectInput{
+    // input parameters
+})
+err := req.Send()
+```
+
+##### V2
+
+```go
+import "context"
+import "github.com/aws/aws-sdk-go-v2/service/s3"
+
+// ...
+
+client : s3.NewFromConfig(cfg)
+
+output, err := client.PutObject(context.TODO(), &s3.PutObjectInput{
+	// input parameters
+})
+```
+
+### Service Data Types
+
+The top-level input and output types of an operation are found in the service client package. The input and output shape
+for a given operation follow the pattern of `<OperationName>Input` and `<OperationName>Output`. Where
```

Comment:
"Where" should not begin a new sentence here but should be a clause in the preceding sentence (change period to comma and lowercase "Where")